README.md

```diff
@@ -83,25 +104,23 @@ for some plugins (e.g.: [`serverless-offline`](https://github.com/dherault/serve
 _Note_: with the defaults above `index.js` will be saved, overwriting without
 warning, in the project root (the parent dir of `out`).
 
-### Configuration
+### Exit on compilation warnings
 
-As an alternative to `cljsCompiler: lumo`, `cljsCompiler.lumo` may be specified
-as a map of options.  Currently supported:
+Lumo generates warnings such as `WARNING: Use of undeclared Var` to signal
```

Comment:
Could you possibly condense this?  I think if the option was named differently, just including it in the example options map is sufficient, and doesn't require an explanation.

If you think it needs to be explained, we should maybe consider moving this stuff to a separate README.md in `serverless_lumo` and linking to it from here, as this file is getting large.

```
  cljsCompiler:
     lumo:
       dependencies:
         - andare:0.7.0
       classpath:
         - /tmp/
       localRepo: /xyz
       cache: /cache | none
       exitOnCompilerWarning: true | false 
```

Reply:
Sure I can do that